customer_operations.py

```diff
@@ -0,0 +1,47 @@
+
+import requests
+
+class CustomerOperations:
+
+    def __init__(self, url="http://localhost:5000"):
+        self.url = url
+
+         
+    def add_customer(self, name="default customer name", phone="default phone number", postal_code=0):
+        query_params = {
+            "name": name,
+            "phone": phone,
+            "postal_code": postal_code
+        }
+        response = requests.post(self.url+"/customers",json=query_params)
```

Comment:
One opportunity for improvement would be to add error handling from the API responses like so.
```suggestion
        response = requests.post(self.url+"/customers",json=query_params)
        if response.code == 200:
            # do normal thing
        if response.code >= 300 and response.code < 500:
            # print error message about bad input. Maybe pull info from response to give more context.
        if response.code >= 500:
            # print generic error message. "Sorry, don't know what went wrong..."
```